### ISSUE TITLE
Use a config instead of hardcoding it in the code

### ISSUE THREAD

**Issue by Taluu** [43/symfony-bot#3]
Currently, there are a lot of "channel id", "group id", "reaction id", .. and so on that are hardcoded.

I think it'd be better to be able to "configure" it, so that the bot could be used on other discords. :D

**Comment by Pierstoval** [comment]
What about using a single file using https://ne-on.org/ ? 😄

**Comment by ricknox** [comment]
This bot is built with in mind that it is for the Symfony Discord server only. Yet I do agree that it must be configurable. I will look at the options and what is easiest to implement.

**Comment by Taluu** [comment]
Yup, and you never know if / when the ids etc would change, so it's be easier to change everything in one go. :}

**Comment by ricknox** [comment]
@Pierstoval I'm not familiar with ne-on. It looks easy but their documentation looks slim. Do you got a example?

**Comment by Pierstoval** [comment]
TL;DR: It's the same thing as Yaml!

=> https://github.com/matej21/neon-js#entity

**Comment by Pierstoval** [comment]
Else, if it's too much for ya, just use json :trollface:

**Comment by Taluu** [comment]
I'd stick to json actually.

**Comment by ricknox** [comment]
I agree, let's stick with a `discord_config.json` file.
